pkg/apis/config/feature_flags_test.go

```diff
@@ -69,9 +69,11 @@ func TestNewFeatureFlagsFromConfigMap(t *testing.T) {
 				VerificationNoMatchPolicy:        config.FailNoMatchPolicy,
 				EnableProvenanceInStatus:         false,
 				ResultExtractionMethod:           "termination-message",
-				MaxResultSize:                    4096,
-				SetSecurityContext:               true,
-				Coschedule:                       config.CoscheduleDisabled,
+				EnableCancelUsingEntrypoint:      true,
+
```

Comment:
nit: can you remove this extra line please?

Reply:
removed